### ISSUE TITLE
redux_gl failing to reduce some alleles

### ISSUE THREAD

**Issue by mmaiers-nmdp** [nmdp-bioinformatics/py-ard#171]
the allele `DQB1*02:02` should be converted to `DQB1*02:01` when calling redux_gl in 'lgx
 mode.

The current release does this correctly:

```
venv/bin/pyard -v 3.48.0 --gl "DQB1*02:02" -r lgx
DQB1*02:01
```

But the allele `DQB1*02:02` has been detected in the input to EM after py-ard reduction to "lgx" so this was not working correctly very recently.

Great that its fixed but this issue is open to try to determine why this bug was there to start with.

**Comment by pbashyal-nmdp** [comment]
Most likely it was reduced with `U2` mode, it reduces to `DQB1*02:02` in `U2` mode.

```
pyard -v 3.48.0 --gl "DQB1*02:02" -r U2
DQB1*02:02
```

**Comment by mmaiers-nmdp** [comment]
This is for HapLogic.  It was definitely not reduced in U2 mode.

The issue seems to a side effect of expansion (using "W") mode and then reduction.

Here is the "lgx" reduction of all alleles returned by `ard.redux_gl("DQB1*02:02","W")`

```
DQB1*02:02:01:01   DQB1*02:01
DQB1*02:02:01:02   DQB1*02:01
DQB1*02:02:01:03   DQB1*02:01
DQB1*02:02:01:04   DQB1*02:01
DQB1*02:02:01:05   DQB1*02:01
DQB1*02:02:01:06   DQB1*02:01
DQB1*02:02:01:07   DQB1*02:01
DQB1*02:02:01:08   DQB1*02:01
DQB1*02:02:01:09   DQB1*02:01
DQB1*02:02:01:10   DQB1*02:01
DQB1*02:02:01:11   DQB1*02:01
DQB1*02:02:01:12   DQB1*02:01
DQB1*02:02:01:13   DQB1*02:01
DQB1*02:02:02   DQB1*02:01
DQB1*02:02:03   DQB1*02:01
DQB1*02:02:04   DQB1*02:02
DQB1*02:02:05   DQB1*02:01
DQB1*02:02:06:02   DQB1*02:01
DQB1*02:02:07   DQB1*02:01
DQB1*02:02:08   DQB1*02:01
DQB1*02:02:09   DQB1*02:01
DQB1*02:02:10   DQB1*02:02
DQB1*02:02:11   DQB1*02:02
DQB1*02:02:12   DQB1*02:01
DQB1*02:02:13   DQB1*02:02
DQB1*02:02:14   DQB1*02:02
DQB1*02:02:15   DQB1*02:02
DQB1*02:02:16   DQB1*02:02
DQB1*02:02:17   DQB1*02:02
DQB1*02:02:18   DQB1*02:01
```

Whats interesting about this is that "ping" fixes this because the alleles that go to `DQB1*02:01` are in the G group but the alleles that go to "DQB1*02:02" are in the P group but not the G group.


### in ping branch

```
% scripts/pyard -v 3.48.0 --gl "DQB1*02:02:17" -r lgx
DQB1*02:01
```

### in master branch
```
$ venv/bin/pyard -v 3.48.0 --gl "DQB1*02:02:17" -r lgx
DQB1*02:02
```

I'll leave this open and resolve it with the ping pull request (working on it now).
I think making ping: True should be the default

**Comment by pbashyal-nmdp** [comment]
Fixed with #178
